lambdaclass/mirra_backend
# 'Self' targeting strategy

Closes [[TargetingStrategy] Self](https://github.com/lambdaclass/champions_of_mirra/issues/406)

## Motivation
Implement new strategy to target the caster unit of a skill.

## Summary of changes
New choose_targets function clause with the 'self' type of targeting strategy, it returns the caster unit id.

## How to test it?

Set the targeting strategy for one of your unit character's skills to "self", you can do so in the skills.json file, running Champions.Config.import_skill_config() to update the running backend. Then run a battle and verify when the modified skill is cast, it targeted the same unit that casted it. You can check battle.md on how to run battles.

## Checklist
